zio-sql/jvm/src/main/scala/Sql.scala

```diff
@@ -185,11 +190,23 @@ trait Sql {
 
     type Aux[A] = Table { type TableType = A }
 
-    sealed trait Source[F[_], A] extends Table {
+    sealed trait Source extends Table {
+      type Repr[_]
+      type Cols
       val name: TableName
-      val columnSchema: ColumnSchema[A]
-      val columns: F[TableType]
-      val columnsUntyped: List[Column.Untyped]
+      val columnSchema: ColumnSchema[Cols]
+      val columns: Repr[TableType]
+    }
+    object Source {
+      type Aux_[F[_], B] = Table.Source {
```

Comment:
This is a bit ugly but couldn't think of another way to have an explicit return type for `Cons.table` (line 104)